src/pages/privacy.md

```diff
@@ -49,7 +49,6 @@ We have checked that our 3rd party solution providers adhere to GDPR guidelines
 - [Ti.to Privacy Policy](https://ti.to/privacy)
 - [PayPal Privacy Policy](https://www.paypal.com/en/webapps/mpp/ua/privacy-full)
 - [Splitbee Privacy Policy](https://splitbee.io/privacy)
-- [Google Privacy Policy](https://policies.google.com/privacy?hl=en-US)
 
 We will not share or sell your personal data to any other 3rd party for their own marketing purposes unless we have your permission to do so.
 
```

Comment:
Should we also modify the `Last updated: 4 March, 2023` to `Last updated: 17 ...` ?